src/lib/network/network_autoyast.rb

```diff
@@ -157,6 +157,8 @@ def configure_hosts(write: false)
       end
       hosts_config = hosts_config.to_h.delete_if { |k, v| k.empty? || v.empty? }
 
+      return false if hosts_config.empty?
```

Comment:
With this fix, the behavior should be the same than in SLE-12-X that is, should fallback to the ResolveHostname... behavior, other fix than that affecting the second stage will be a real change of behavior.